Earthfile

```diff
@@ -561,6 +561,8 @@ dind:
         # in order to display the upstream-version, e.g. "24.0.5-1".
         SET DOCKER_VERSION_TAG="$(echo $DOCKER_VERSION | sed 's/^[0-9]*:\([^~]*\).*$/\1/')"
         RUN if echo $DOCKER_VERSION_TAG | grep "[^0-9.-]"; then echo "DOCKER_VERSION_TAG looks bad; got $DOCKER_VERSION_TAG" && exit 1; fi
+    ELSE IF [ "$OS_IMAGE" = "alpine" ]
+        RUN apk add iptables-legacy # required for older kernels
```

Comment:
I know this would probably be harder to maintain but I wonder if we should try to pin to a specific version to ensure to avoid surprises (unless the fact it's a legacy version means it's not going to get anymore updates?)

Reply:
there might still be updates to the iptables-legacy package, but they should be backwards compatable, pinning to a specific version here seems overkill -- once this image is built and pushed, we shouldn't need to continually rebuild it.

when we do rebuild it (e.g. a new docker version, which is pinned), we'll need to do more testing.

overall the dind process needs some improvements, which I quickly highlighted in #3786

I'm inclined to leave this as-is while we sort out the immediate wsl issue.